api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -133,6 +134,20 @@ public static void set(Supplier<OpenTelemetry> supplier) {
     }
   }
 
+  /**
+   * Evaluate if the global instance has been set without the side effects of {@link #get()}.
+   *
+   * <p>This is useful for evaluating if any code, like the OpenTelemetry javaagent, has previously
+   * set the global instance while preserving the ability to set it.
+   *
+   * @return true if the global instance has been set, false otherwise.
+   */
+  public static boolean isSet() {
```

Comment:
a couple of other options for naming, in case either of them jump out to you:

- `isInitialized()`
- `hasBeenSet()`

Reply:
I like `isSet` - easiest to discover